src/sweeper/address_parser.py

```diff
@@ -148,6 +162,15 @@ def normalize_street_type(type_text):
     raise InvalidStreetTypeError(type_text)
 
 
+HWY_REGEX = compile('(SR|STATE ROUTE|HIGHWAY)')
```

Comment:
i think this should be compiled at the import level. This will be compiled at the class level now if i'm reading this right. I don't think that will give any perf boost. I think here, it would actually decrease perf.

Reply:
I think route and rt need to be handled also right @ZachBeck

Reply:
I think it might be good. I think you'll always be looking for State Route and never Route by itself (if I'm reading it right).

Reply:
This is compiled at the import level outside of the Address class.

Reply:
I am lonely king at user data with route 63 without the state piece...

Reply:
what about cases like Indian Route 5061 or Ridge Route Rd?

Reply:
I think that `Route 63` should be left alone, no? There are road names that are not highways named "Route", aren't there?

As for @ZachBeck's cases, I think that we are good:
```
>>> Address('12 Indian Route 5061')
Parsed Address:
{'address_number': '12',
 'normalized': '12 INDIAN ROUTE 5061',
 'street_name': 'INDIAN ROUTE 5061'}
>>> Address('12 Ridge Route Rd')
Parsed Address:
{'address_number': '12',
 'normalized': '12 RIDGE ROUTE RD',
 'street_name': 'RIDGE ROUTE',
 'street_type': 'RD'}
```

Reply:
I think we should move this to the top. It's confusing where it is placed